ignite/src/test/resources/log4j.properties

```diff
@@ -8,8 +8,8 @@
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.Target=System.out
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
-log4j.appender.stdout.layout.ConversionPattern=%d{ABSOLUTE} %5p %c{1}:%L - %m%n
+log4j.appender.stdout.layout.ConversionPattern=%d{ABSOLUTE} %5p %c{1}:%M:%L - %m%n
 
 log4j.rootLogger=info, stdout
 
-log4j.logger.org.hibernate=info
+log4j.logger.org.hibernate=debug
```

Comment:
This changes should be in a separate commit, you are only changing the properties of the log file

Reply:
You right.  I will restore it. It needs for development and testing

Reply:
fixed